Makefile.am

```diff
@@ -295,6 +295,47 @@ libeos_updater_flatpak_installer_libeos_updater_flatpak_installer_@EUFI_API_VERS
 # eos-updater-flatpak-installer program
 libexec_PROGRAMS += eos-updater-flatpak-installer/eos-updater-flatpak-installer
 
+###############################################################################
+# Introspection
+-include $(INTROSPECTION_MAKEFILE)
+INTROSPECTION_GIRS =
+INTROSPECTION_SCANNER_ARGS =
+INTROSPECTION_COMPILER_ARGS =
+AM_DISTCHECK_CONFIGURE_FLAGS += --enable-introspection
+EXTRA_DIST += m4/introspection.m4
+
+if HAVE_INTROSPECTION
+libeos-updater-flatpak-installer/Libeufi-@EUFI_API_VERSION@.gir: libeos-updater-flatpak-installer/libeos-updater-flatpak-installer-@EUFI_API_VERSION@.la
```

Comment:
By convention, `.gir` files don’t have a `Lib` prefix — so this should probably be `EosUpdaterFlatpakInstaller-@EUFI_API_VERSION@.gir`.

Reply:
For introspection to work, the library needs to be linkable against, which means you need to change `noinst_LTLIBRARIES += libeos-updater-flatpak-installer/libeos-updater-flatpak-installer-@EUFI_API_VERSION@.la` above to `lib_LTLIBRARIES += libeos-updater-flatpak-installer/libeos-updater-flatpak-installer-@EUFI_API_VERSION@.la`.

Reply:
> By convention, …

Ah, it seems you already did that in the follow up commit. :+1: